test/include.test.js

```diff
@@ -15,14 +15,14 @@ describe('include option', function () {
     Post = ds.createModel('post', { title: String });
     app.model(Post);
     Post.customMethod = function (cb) {
-      cb(null, {prop: 'value'});
+      cb(null, {prop: 'value', id: null});
```

Comment:
I think this is ok. Might be a good idea to document the expectation that an id must be set